Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -321,7 +321,7 @@ class TabLocationView: UIView {
 
     fileprivate func updateTextWithURL() {
         (urlTextField as? DisplayTextField)?.hostString = url?.host ?? ""
-        urlTextField.text = url?.schemelessAbsoluteString.trim("/")
+        urlTextField.text = url?.domainURL.schemelessAbsoluteString.trim("/")
```

Comment:
This `domainURL` strips `m.` and `mobile.` schemes as well.
I talked to security team and they want to hide scheme only with `www.` case.

So one additional helper method needs to be added to `URLExtensions.swift` to remove only the `www` part